src/amo/sagas/home.js

```diff
@@ -136,6 +142,7 @@ export function* fetchHomeData({
       });
     } catch (error) {
       log.warn(`Home add-ons failed to load: ${error}`);
+      yield put(abortFetchHomeData());
```

Comment:
You don't need this because the error will get caught in an outer block which already aborts the fetch. It would be good to keep the extra test for it, though.